aiidalab_widgets_base/viewers.py

```diff
@@ -233,7 +234,7 @@ def atoms_in_representaion(self, structure: ase.Atoms | None = None):
         if structure:
             if self.uuid in structure.arrays:
                 return structure.arrays[self.uuid] >= self.atom_show_threshold
```

Comment:
Not from this PR, but since you are here. 
Because python if..else if short-circuting, I think this condition can be simplified to:

```suggestion
        if structure and self.uuid in structure.arrays:
            return structure.arrays[self.uuid] >= self.atom_show_threshold
```